compact/tof_endcap.xml

```diff
@@ -39,8 +39,8 @@
     <comment> 1 um padding to not have layer and module touch (ACTS requirement) </comment>
     <constant name="TOFEndcapLayer_thickness"   value="TOFEndcapMod_thickness + 2 * TOFEndcapMod_dz + 1 * um" />
 
-    <constant name="ForwardTOFMod_rmin"         value="ForwardTOFRegion_tan * ForwardTOF_zmin" />
-    <constant name="ForwardTOFMod_rmax"         value="ForwardTOFRegion_rmax" />
+    <constant name="ForwardTOFMod_rmin"         value="12*cm" />
+    <constant name="ForwardTOFMod_rmax"         value="85*cm" />
```

Comment:
Please do not change parametrization with hard-coded values. If the values in the parametrization are incorrect, they should be adjusted there. @sly2j may comment.